### PR TITLE
Fix links in docs

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

**Review by amityadav0 — COMMENTED** [review]
Please update the links in examples as well. [setup](../README.md) etc.

**Comment by ratik21** [comment]
> Please update the links in examples as well. [setup](../README.md) etc.

I did a sweep across User Docs, Algob API docs, Runtime API docs to check for broken links.
Checking examples as well.

File: packages/runtime/README.md [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+- [AccountStore](https://github.com/scale-it/algo-builder/blob/master/packages/runtime/src/account.ts): user can create new accounts using a `AccountStore` object. All information about an account (`apps`, `assets`, `localState`, `globalState` etc..) is stored in `AccountStore`.
+- [Parser](https://github.com/scale-it/algo-builder/tree/master/packages/runtime/src/parser): reads TEAL code and converts it to a list of opcodes which are executable by the interpreter. If a teal code contains an invalid opcode/data, parser will throw an error.
+- [Interpreter](https://github.com/scale-it/algo-builder/tree/master/packages/runtime/src/interpreter): executes a list of opcodes returned by parser. Interpreter creates teal execution stack and interacts with runtime to perform changes related a TEAL code execution. At the end of execution, if the stack contains a single non-zero uint64 element then the teal code is approved, and transaction can be executed.
```
[the remark this answers is not in the document]
Comment (ratik21) [reply packages/runtime/README.md:9]: yes, let's update to absolute links.

File: examples/permissioned-token-freezing/README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ To implement this solution, three basic operations are required in the stateful
 
 ### Setup
 
-Please follow the [setup](../README.md) instructions to install dependencies and update the config. Then we can deploy the asset and smart contracts.
+Please follow the [setup](https://github.com/scale-it/algo-builder/blob/master/examples/README.md) instructions to install dependencies and update the config. Then we can deploy the asset and smart contracts.
```
Comment (robert-zaremba) [inline examples/permissioned-token-freezing/README.md:31]: This is not deployed - so I think this could stays as it was.

**Review by robert-zaremba — APPROVED** [review]
We can consider keeping a relative link to examples main README file, instead putting a full link to github.

File: examples/permissioned-token-freezing/README.md (hunk 2 above)
Comment (ratik21) [reply examples/permissioned-token-freezing/README.md:31]: OK

**Pull request merged**